src/Storefront/Resources/app/storefront/src/main.js

```diff
@@ -163,13 +179,13 @@ document.addEventListener('DOMContentLoaded', () => {
      * This leads to console errors in some browsers, if an element within the modal still has focus.
      */
     const modals = document.querySelectorAll('.modal');
-    modals.forEach((modal) => {
+    for (const modal of modals) {
```

Comment:
was the listener buggy with forEach?

Reply:
nope ... I can also revert that, but biome is always telling me that forEach is slow ...

Reply:
Ok, I think unless there very many modals in the DOM the performance impact should be negligible, but fine for me. 👍